src/ResourceManagement/ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "1.1.0-preview",
+  "version": "1.2.0",
```

Comment:
Our service is going GA as of this 2017-03-01 api-version, do we still keep the SDK as preview?